dfyconfig.toml

```diff
@@ -2,4 +2,4 @@ includes = ["src/**/*.dfy"]
 excludes = []
 
 [options]
-enforce-determinism = true
+enforce-determinism = false
```

Comment:
Why do we need to do this?

Reply:
As discussed offline, this is a consequence of me using `var item :| item in items;` in `Tests.dfy`.